tests/unit/test_endpoint.py

```diff
@@ -140,27 +140,6 @@ def setUp(self):
         super(TestRetryInterface, self).setUp()
         self.retried_on_exception = None
 
-    def max_attempts_retry_handler(self, attempts, **kwargs):
```

Comment:
No that is fine. We can keep them here. Figured to ask because it was not clear.